Api/IndexBuilder/IndexBuilderInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Api\IndexBuilder;
+
+interface IndexBuilderInterface
+{
+    public function buildIndex(int $storeId, ?array $ids, ?array $options): void;
```

Comment:
@damcou Now that I'm looking at this method signature again I'm feeling like we should clarify the second argument to distinguish it from `$storeId` - maybe something like `$entityIds`? WDYT